scripts/V275__INCLUSAO_PARAMETROS_DE_QUEM_RECEBE_PENDENCIA.sql

```diff
@@ -1,60 +1,34 @@
 insert  into  
 	public.parametros_sistema (nome,descricao,valor,ano,ativo,criado_em,criado_por, criado_rf,tipo)
 select
-	'GerarPendenciaAulasDiasNaoLetivos','Gerar pendencia de aulas em dias não letivos para o professor','Professor', 2020, true, now(),'Carga Inicial','Carga Inicial',35
-where
-	not exists(
-	select 	1
-	from public.parametros_sistema 
-	where tipo = 35);
```

Comment:
dessa forma pode duplicar os parametros caso rode o script 2x.
Pra resolver o problema pode colocar where tipo = 35 and valor = 'Professor'